data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -154,6 +154,19 @@ expedition2 = {
    )
 }
 
+expedition2a = {
+   position = "topright",
+   title = _"I am going on a trip and I am taking … with me.",
+   body = (
+      h1(_"Additional Items") ..
+      p(_[[During an expedition, we might need some more wares then just the ones to build a new port.]]) ..
```

Comment:
This sounds confusing as these wares are needed *after* the expedition rather than during it. Suggestion:
```suggestion
      p(_[[It might be useful to take some more wares with us than just the ones to build a new port. That way we will be able to start constructing new buildings as soon as the port has been constructed.]]) ..
```